site/public/js/docker_interface.js

```diff
@@ -64,6 +66,40 @@ function addFieldOnChange() {
     }
 }
 
+function confirmationdialog(url,id,capability) {
+    if (confirm(`Are you sure you want to remove ${id} image?`)) {
+        removeImage(url,id,capability);
+    }
+}
+
+function removeImage(url,id,capability) {
+    $.ajax({
```

Comment:
You could the `fetch` api which is a little bit easier to use especially since we're not using any ajax/xhr specific features 

```js
const response = await fetch(url, {
   'method' : 'POST',
   'body' : JSON.stringify({
        'image' : id,
        'capability' : capability,
        'csrf_token' : csrfToken
    }),
    headers: {
        'Content-Type': 'application/json'
    },
});

response.json()
.then((json) => { //handle response })
.catch((err) => { //handle fetch fail });
 
```

Or something like that, see the docs here
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
@shailpatels I'm having trouble with implementing the fetch feature as I'm receiving a 302 response. I've attempted to modify the CORS mode and the JSONResponse backend, but I haven't been able to resolve the issue. Can you please offer any suggestions or guidance on how to resolve this problem?